modules/oe_translation_epoetry/oe_translation_epoetry.module

```diff
@@ -133,21 +131,3 @@ function oe_translation_epoetry_oe_translation_request_view_alter(array &$build,
     '#url' => $url,
   ];
 }
-
-/**
- * Implements hook_ENTITY_TYPE_presave().
- */
-function oe_translation_epoetry_oe_translation_request_presave(TranslationRequestInterface $entity) {
```

Comment:
i deleted this in favor of the request statuses being set directly in the notification subscriber